Sources/SemanticIndex/UpdateIndexStoreTaskDescription.swift

```diff
@@ -299,7 +298,7 @@ private func adjustSwiftCompilerArgumentsForIndexStoreUpdate(
     // batch mode is not compatible with -index-file
     "-disable-batch-mode",
     // Fake an output path so that we get a different unit file for every Swift file we background index
-    "-o", fileToIndex.pseudoPath + ".o",
+    "-index-unit-output-path", fileToIndex.pseudoPath + ".o",
```

Comment:
FWIW this doesn't reallly matter. -index-unit-output-path only exists because we need it to be different to -o during a build. But for just background indexing, -o is fine.